operators/endpointmetrics/controllers/observabilityendpoint/testdata/crd/hostedclusters.hypershift.yaml

```diff
@@ -0,0 +1,7981 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+  creationTimestamp: "2023-09-20T18:09:38Z"
+  generation: 4
+  name: hostedclusters.hypershift.openshift.io
+  uid: cc3bc7a2-fa66-4bca-93b2-1643bfe3e61d
```

Comment:
Is there some context around why we need these CRDs and under what circumstance? 
In any case we likely would not want to hardcode a UUID in the resources themselves. Same with timestamp and generation info etc.

Reply:
I see now its in the testdata directory which is fine and makes sense. Best to exclude the specifics that will be generated in any case

Reply:
yes I added them for the integration tests with envtest. I have encapsulated the CRDs dependencies here for the addon. Some are already declared at some other places in the codebase. There is some duplication, but it is easier to manage in my view. There are probably better solutions for this.

Reply:
cf https://github.com/stolostron/multicluster-observability-operator/pull/1352#discussion_r1539232751